package/firmware/ipq-wifi/Makefile

```diff
@@ -114,14 +114,16 @@ Do not install it for any other device!
 endef
 
 # Add board name to ALLWIFIBOARDS
-# Place files in this directory as board-<devicename>.<qca4019|qca9888|qca9889|qca9984|qca99x0|ipq8074>
+# Place files in this directory as board-<devicename>.<qca4019|qca9888|qca9889|qca9984|qca99x0|ipq8074|ipq8174>
 # Add $(eval $(call generate-ipq-wifi-package,<devicename>,<display name>))
 
 $(eval $(call generate-ipq-wifi-package,buffalo_wxr-5950ax12,Buffalo WXR-5950AX12))
 $(eval $(call generate-ipq-wifi-package,compex_wpq873,Compex WPQ-873))
 $(eval $(call generate-ipq-wifi-package,dynalink_dl-wrx36,Dynalink DL-WRX36))
 $(eval $(call generate-ipq-wifi-package,edgecore_eap102,Edgecore EAP102))
 $(eval $(call generate-ipq-wifi-package,edimax_cax1800,Edimax CAX1800))
+$(eval $(call generate-ipq-wifi-package,linksys_mx4200v1,Linksys MX4200))
+$(eval $(call generate-ipq-wifi-package,linksys_mx4200v2,Linksys MX4200))
```

Comment:
Do these BDFs differ at all?

Reply:
I haven't actually tested that, but I don't think they differ

Reply:
Wait, where are these even coming from?
I am not seeing them in the BDF repo, nor I am seeing a PR for them.

Please binary diff the stock BDF-s first, and then package them.

Reply:
yes i was about to create a pull request add the BDFs

Reply:
done

Reply:
But you are still not even generating the meta package.

You need to add your device to:
https://github.com/openwrt/openwrt/blob/main/package/firmware/ipq-wifi/Makefile#L30

Reply:
some how i missed that, ive added it now